.buildkite/serverless_integration_pipeline.yml

```diff
@@ -16,8 +16,9 @@ steps:
     command: ./.buildkite/scripts/setup_java.sh && ./ci/serverless/elastic_integration_filter_tests.sh
   - label: "central pipeline management test"
     command: ./.buildkite/scripts/setup_java.sh && ./ci/serverless/cpm_tests.sh
-  - label: "Logstash legacy monitoring test"
-    command: ./.buildkite/scripts/setup_java.sh && ./ci/serverless/monitoring_tests.sh
+# Legacy monitoring is disabled. Serverless does not support /_monitoring/bulk, hence the test always fails to ingest metrics.
+#  - label: "Logstash legacy monitoring test"
+#    command: ./.buildkite/scripts/setup_java.sh && ./ci/serverless/monitoring_tests.sh
```

Comment:
Looking at https://github.com/buildkite/feedback/issues/55#issuecomment-305028566 we should be able to just skip it:

```suggestion
# Legacy monitoring is disabled. Serverless does not support /_monitoring/bulk, hence the test always fails to ingest metrics.
- label: "Logstash legacy monitoring test"
  command: ./.buildkite/scripts/setup_java.sh && ./ci/serverless/monitoring_tests.sh
  skip: true
```